src/directory/directory.mjs

```diff
@@ -269,6 +269,9 @@ export const directory = {
                     {
                       path: 'src/pages/[platform]/build-a-backend/functions/examples/create_user_profile_record/index.mdx'
                     },
+                    {
+                      path: 'src/pages/[platform]/build-a-backend/functions/examples/override_token/index.mdx'
```

Comment:
I just noticed the casing on these two examples is different, can we change this to kebab-case to align with the others?